crates/router/src/connector/trustpay/transformers.rs

```diff
@@ -885,6 +910,18 @@ fn handle_cards_refund_response(
     Ok((error, refund_response_data))
 }
 
+fn handle_webhooks_refund_response(
+    response: WebhookPaymentInformation,
+) -> CustomResult<(Option<types::ErrorResponse>, types::RefundsResponseData), errors::ConnectorError>
+{
+    let refund_status = storage_models::enums::RefundStatus::try_from(response.status)?;
+    let refund_response_data = types::RefundsResponseData {
+        connector_refund_id: "".to_string(),
```

Comment:
Add comment why this is empty and check how merchant will map this refund in connector dashboard

Reply:
In webhooks payload, I am not getting connector_refund_id

Reply:
if we are not getting the connector_refund_id then, think about a way to handle this.

This update the connector_refund_id as empty string in database?